frontend/app/components/wp-panels/watchers-panel/watchers-panel.directive.html

```diff
@@ -8,9 +8,23 @@
   <div ng-hide="vm.error">
     <div class="work-package--watchers work-package--details--long-field"
          ng-class="{'-read-only': !vm.allowedToRemove}">
-      <work-package-watcher watcher="watcher"
-                            ng-repeat="watcher in vm.watching track by watcher.id"></work-package-watcher>
+      <wp-watcher watcher="watcher"
+                  ng-repeat="watcher in vm.watching track by watcher.id"></wp-watcher>
+    </div>
+    <div class="work-package--watchers-lookup" ng-if="vm.allowedToAdd">
+      <form name="watcherForm" novalidate>
+        <input type="text"
+               class="wp-watcher--autocomplete typeahead-autocomplete"
+               ng-class="{ '-error': vm.autocompleteInput.length && noResults }"
+               ng-model="vm.autocompleteInput"
+               op-fullwidth-typeahead
+               typeahead-is-open="autocompleteIsOpen"
+               typeahead-on-select="vm.addWatcher($model)"
+               typeahead-no-results="noResults"
+               typeahead-wait-ms="100"
+               placeholder="{{ ::vm.text.autocomplete.placeholder }}"
+               uib-typeahead="item as item.name for item in vm.autocompleteWatchers($viewValue)">
```

Comment:
Is it possible to add the avatar to the typeahead? It's a little strange that there are avatars for the already selected watchers but not for the candidates:

![image](https://cloud.githubusercontent.com/assets/617519/20308982/7fd4b35a-ab46-11e6-838e-4b2b3b511ea2.png)